spacy/lang/nl/syntax_iterators.py

```diff
@@ -0,0 +1,75 @@
+from typing import Union, Iterator
+
+from ...symbols import NOUN, PRON
+from ...errors import Errors
+from ...tokens import Doc, Span
+
+
+def noun_chunks(doclike: Union[Doc, Span]) -> Iterator[Span]:
+    """
+    Detect base noun phrases from a dependency parse. Works on Doc and Span.
+    The definition is inspired by https://www.nltk.org/book/ch07.html
```

Comment:
Did you only use the definition, or also the reference implementation from NLTK?

Reply:
I used the definition for my reference, but I did not have a look at the NLTK implementation.